PleaseGiftMeInPerson/PleaseGiftMeInPerson.cs

```diff
@@ -65,8 +65,11 @@ public override void Entry(IModHelper helper)
 					? Game1.content.Load<Dictionary<string, string>>("Data/AntiSocialNPCs")
 					: new();
 
+				foreach (KeyValuePair<string, string> kv in npcDispositions)
+					if (kv.Value.Split('/').Length != 12)
+						this.Monitor.Log($"{kv.Key.ToString()}: {kv.Value.ToString()} is missing elements!", LogLevel.Warn);
 				var characters = npcDispositions
-					.Select(c => (name: c.Key, displayName: c.Value.Split('/')[11]))
+					.Select(c => (name: c.Key, displayName: c.Value.Split('/').Last()))
```

Comment:
I don't think using the last element is a good idea - if someone puts more elements than are actually used, this will be wrong; this will also be wrong in case of missing elements. I think the only action we could possibly take here is to use `c.Key` instead.
Also, any particular mods which have their NPC dispositions broken like that? This sounds like something that shouldn't really be supported and the mods adding those dispositions should be fixed instead,

Reply:
It was an issue in https://www.nexusmods.com/stardewvalley/mods/9271 for SV 1.7 (Marlon only had 11 elements) but I still think it's better to warn about the bad config than to crash yourself and stop working?